src/hotspot/share/memory/virtualspace.cpp

```diff
@@ -32,6 +32,7 @@
 #include "runtime/globals_extension.hpp"
 #include "runtime/java.hpp"
 #include "runtime/os.hpp"
+#include "runtime/os.inline.hpp"
```

Comment:
Note that including `os.hpp` is no longer needed when you include `os.inline.hpp`.